PowerEditor/installer/functionList/overrideMap.xml

```diff
@@ -97,7 +97,8 @@
 			<association id= "visualprolog.xml"  langID= "84"/>
 			<association id= "typescript.xml"  	 langID= "85"/>
 			<association id= "gdscript.xml"  	 langID= "88"/>
-
+			<association id= "hollywood.xml"  	 langID= "89"/>
```

Comment:
Please modify the above 2 lines to:
```xml
<association id= "gdscript.xml"  	 langID= "87"/>
<association id= "hollywood.xml"  	 langID= "88"/>
```

Reply:
@donho 
> Please modify the above 2 lines to:
> 
> ```
> <association id= "gdscript.xml"  	 langID= "87"/>
> <association id= "hollywood.xml"  	 langID= "88"/>
> ```

Are you sure? I think my version is correct because `L_GDSCRIPT` is 88 and `L_HOLLYWOOD` is 89 so that's why they should use 88 and 89 in the XMLs as well, shouldn't they? Also note that `gdscript.xml` uses 88 in master.

Reply:
@afalkenhahn 
OK. I will merge this PR and fix it if it's really needed.
Thank you for the quick fix.